tests/test_wave_03.py

```diff
@@ -54,13 +55,16 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
+    assert INTRIGUE_3 in friends_unique_movies
+    assert HORROR_1 in friends_unique_movies
+    assert FANTASY_4 in friends_unique_movies
```

Comment:
👍 Here as well, we explored the input data, reasoned about what should be in hte result ourselves, then wrote the checks to ensure that the 3 movies in the liist are the expected ones.